firefox-ios/firefox-ios-tests/Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -214,7 +214,10 @@ class TrackingProtectionTests: BaseTestCase {
             timeout: TIMEOUT_LONG
         )
         mozWaitForElementToExist(app.staticTexts.elementContainingText("Firefox has not connected to this website."))
-        XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "Connection not secure")
+
+        // The lock icon is no longer here.
+        // https://github.com/mozilla-mobile/firefox-ios/issues/22600
+        // XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "Connection not secure")
```

Comment:
This test fails currently because the lock icon is no longer here. Need some clarification from the devs.

Reply:
Have you got any responses?

Reply:
Not yet. I will bring up the issue during bug triage.